airdrop/infrastructure/repositories/airdrop_repository.py

```diff
@@ -238,7 +238,7 @@ def get_airdrop_window_claimable_info(self, airdrop_id, airdrop_window_id, user_
 
     def fetch_total_rewards_amount(self, airdrop_id, address):
         try:
-            query = text("select sum(ur.rewards_awarded) AS 'total_rewards' FROM user_rewards ur, airdrop_window aw where ur.airdrop_window_id = aw.row_id and ur.address = :address and aw.airdrop_id = :airdrop_id and aw.claim_start_period <= current_timestamp and ur.airdrop_window_id not in (select airdrop_window_id from claim_history where address = :address and transaction_status in ('SUCCESS', 'PENDING')) and ur.airdrop_window_id > (select ifnull (max(airdrop_window_id), -1) from claim_history where address = :address and transaction_status in ('SUCCESS', 'PENDING'));")
+            query = text("select  sum(ur.rewards_awarded) AS 'total_rewards' FROM  user_rewards ur,  airdrop_window aw where  ur.airdrop_window_id = aw.row_id  and ur.address = :address  and aw.airdrop_id = :airdrop_id  and aw.claim_start_period <= current_timestamp  and exists ( select 1 from airdrop_window aw3 where current_timestamp <= aw3.claim_end_period  and aw3.airdrop_id = :airdrop_id and aw3.claim_start_period <= current_timestamp )  and ur.airdrop_window_id > (  select ifnull (max(airdrop_window_id), -1)  from claim_history  where address = :address and transaction_status in ('SUCCESS', 'PENDING')) and ur.airdrop_window_id in (  SELECT airdrop_window_id  FROM user_registrations ur2  WHERE ur2.address = :address and ur2.airdrop_window_id in ( select  row_id from  airdrop_window aw2 where  aw2.airdrop_id = :airdrop_id));")
```

Comment:
Lets remove aliases when not necessary.
query = text("select  sum(ur.rewards_awarded) AS 'total_rewards' FROM  user_rewards ur,  airdrop_window aw where  ur.airdrop_window_id = aw.row_id  and ur.address = :address  and aw.airdrop_id = :airdrop_id  and aw.claim_start_period <= current_timestamp  and exists ( select 1 from airdrop_window where current_timestamp <= claim_end_period  and airdrop_id = :airdrop_id and claim_start_period <= current_timestamp )  and ur.airdrop_window_id > (  select ifnull (max(airdrop_window_id), -1)  from claim_history  where address = :address and transaction_status in ('SUCCESS', 'PENDING')) and ur.airdrop_window_id in (  SELECT airdrop_window_id  FROM user_registrations  WHERE address = :address and airdrop_window_id in ( select  row_id from  airdrop_window where  airdrop_id = :airdrop_id));")